pvlib/temperature.py

```diff
@@ -978,3 +978,397 @@ def prilliman(temp_cell, wind_speed, unit_mass=11.1, coefficients=None):
+        wind_adj = 0.51
+        u_noct = 800.0 * self.alpha / (noct - 20.0)
+        self.u_const = u_noct * 0.6
+        self.du_wind = u_noct * 0.4 * wind_adj
+        return self
+
+    def to_noct_sam(self):
+        '''
+        Convert the generic model parameters to NOCT SAM model equivalents.
+
+        Returns
+        ----------
+        model_parameters : dict
+            See :py:func:`pvlib.temperature.noct_sam` for
+            model parameter details.
+        '''
+        # NOCT is determined with wind speed near module height
+        # the adjustment reduces the wind coefficient for use with 10m wind
+        wind_adj = 0.51
+        u_noct = self.u_const + self.du_wind / wind_adj
+        noct = 20.0 + (800.0 * self.alpha) / u_noct
+        return dict(noct=noct,
+                    module_efficiency=self.eta,
+                    transmittance_absorptance=self.alpha)
+
+    def use_sapm(self, a, b, wind_fit_low=1.4, wind_fit_high=5.4):
+        '''
+        Use the SAPM module model parameters to set the generic equivalents.
+
+        In the SAPM the heat transfer coefficient increases exponentially
+        with windspeed, whereas in the other models the increase is linear.
+        This function equates the generic linear model to SAPM at two
+        specified winds speeds, thereby defining a linear approximation
+        for the exponential behavior.
+
+        Parameters
+        ----------
+        a, b : float
+            See :py:func:`pvlib.temperature.noct_sam` for details.
+
+        wind_fit_low, wind_fit_high : float, optional
+            Two wind speed values at which the generic linear model
+            must be equal to the SAPM model. The default
+            values represent wind speed at 10m height. [m/s]
+        '''
+        u_low = 1.0 / np.exp(a + b * wind_fit_low)
+        u_high = 1.0 / np.exp(a + b * wind_fit_high)
+
+        du_wind = (u_high - u_low) / (wind_fit_high - wind_fit_low)
+        u_const = u_low - du_wind * wind_fit_low
+
+        net_absorptance = self.alpha - self.eta
+        self.u_const = u_const * net_absorptance
+        self.du_wind = du_wind * net_absorptance
+        return self
+
+    def to_sapm(self, wind_fit_low=1.4, wind_fit_high=5.4):
+        '''
+        Convert the generic model parameters to SAPM model equivalents.
+
+        In the SAPM the heat transfer coefficient increases exponentially
+        with windspeed, whereas in the other models the increase is linear.
+        This function equates SAPM to the generic linear model at two
+        specified winds speeds, thereby defining an exponential approximation
+        for the linear behavior.
+
+        Parameters
+        ----------
+        wind_fit_low, wind_fit_high : float, optional
+            Two wind speed values at which the generic linear model
+            must be equal to the SAPM model. The default
+            values represent wind speed at 10m height. [m/s]
```

Comment:
I didn't know sphinx would do this, but apparently it lists grouped parameters separately and makes a copy of the description for each (see [here](https://pvlib-python--1463.org.readthedocs.build/en/1463/reference/generated/pvlib.temperature.GenericLinearModel.to_sapm.html)).  So right now both `wind_fit_low` and `wind_fit_high` are each described as "two values".  Could rewrite this description or split out the parameters into separate entries.

And same for `use_sapm`, of course.